### ISSUE TITLE
outdated plugin

### ISSUE THREAD

**Issue by Jonathannavaxd** [MW90-NL/httpeventserver#1]
this plugin is outdated friend

**Comment by MW90-NL** [comment]
Hi Jonath, what are you using this for, and are you saying it isn't working anymore due to being outdated.
I haven't played OSRS for a while

**Comment by MW90-NL** [comment]
Didn't mean to close the issue

**Comment by NathanDunahoo** [comment]
![image](https://github.com/MW90-NL/httpeventserver/assets/49346923/24013bf2-d04f-4a51-a3bd-822e19652c6b)

**Comment by Thurdles** [comment]
Yeah, having the same issue of it being outdated. OSRS just had a big update a couple weeks ago and since then it has not been working. Trying to use this plugin to get player position data and minimap data.

**Comment by Dwayne-Zammit** [comment]
Hi,

Just a tip to fix for the creator:
I believe you can comment the below code to fix or find an alternative solution:
path:```runeliteclient\src\main\java\net\runelite\client\plugins\httpeventserver\HttpEventServerPlugin.java```

code: 
```
camera.addProperty("x2", client.getCameraX2());
camera.addProperty("y2", client.getCameraY2());
camera.addProperty("z2", client.getCameraZ2()); 
```
Regards

**Comment by AranWeld** [comment]
Hi MW90, same issue, can't install the plugin anymore as it is incompatible with RL. Thanks

**Comment by MW90-NL** [comment]
Thanks all for reach out and raising this, I will look into this over the weekend

Would love to hear how people are using the plugin!

**Comment by AranWeld** [comment]
I used a python script to read the current HP values, and then send myself a text message (through OS terminal) when HP went below 20, so that I could do NMZ even more AFK.  I also used the coordinates to text me when I've exited NMZ.  That way I could leave my PC running for hours and do housework and only run up to check when I needed to!

**Comment by Dwayne-Zammit** [comment]
Hi @MW90-NL , Plugin is not visible in Runelite plugins...
I tried compiling Runelite myself and loading this plugin, it still works... Any reason why not available in Plugin hub please?
Could you perhaps add it again?

**Comment by MW90-NL** [comment]
@Dwayne-Zammit @AranWeld @Thurdles @NathanDunahoo @Jonathannavaxd 

Sorry, I fixed the issues but this plugin is no-longer allowed under Jagex's / Runelite policys

https://github.com/runelite/plugin-hub/pull/9457
